Makefile

```diff
@@ -8,40 +6,38 @@ else
 endif
 
 TARGETS=awesome
-BUILDLN=build
+BUILDDIR=build
 
-all: $(TARGETS) $(BUILDLN) ;
+all: $(TARGETS) ;
```

Comment:
And why does it make a difference what `all` depends on? If `all` depends on `./build`, then `all` needs to run if `build` does not exist. However, as I indicated in the previous comment, that's the case anyway. If `build` exists, then that dependency makes no difference.